libpod/container_internal.go

```diff
@@ -945,13 +945,24 @@ func (c *Container) generateSpec(ctx context.Context) (*spec.Spec, error) {
 		}
 	}
 
+	var (
+		uid       uint32
+		gid       uint32
+		userIsInt bool
+	)
 	if c.config.User != "" {
 		if !c.state.Mounted {
 			return nil, errors.Wrapf(ErrCtrStateInvalid, "container %s must be mounted in order to translate User field", c.ID())
 		}
-		uid, gid, err := chrootuser.GetUser(c.state.Mountpoint, c.config.User)
-		if err != nil {
-			return nil, err
+		// if user is an int, don't look it up in /etc/passwd, just the the ProcessUID to the int given
```

Comment:
"the the" (I'm not sure what the comment is meant to be)

Reply:
fixed